kpi/docs/api/v2/asset_usage/list.md

```diff
@@ -0,0 +1 @@
+## List all asset usage of a user
```

Comment:
Even if it's the `list` action, that's a trap 🤪! It's not a list. It is a just a dictionary. Title should be something like "Get user's asset usage" .